.github/workflows/nightly.yml

```diff
@@ -33,8 +33,7 @@ jobs:
         env:
           SHA_EXT: sha256sum.txt
         run: |
-          filename=$(find pkg-build/cpack_output -type f \( -iname \*.deb -o -iname \*.rpm \))
```

Comment:
seems merge calculate the SHA check sum into one line?

Reply:
Yes